From adba87bd5ad562b0d62b5f23925429ace4ef3c5b Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 8 Dec 2021 22:47:37 +0100
Subject: [PATCH] rescue-image: gen_initramfs_list.sh was renamed to
 gen_initramfs.sh

This is the upstream change in Linux Kernel Build repository.
Refer: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/usr/Makefile?h=v5.10.74&id=80e715a06c2d0a24dbb818bb1dba2ef0f20d7068
---
 hardware/rescue-image/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hardware/rescue-image/Makefile b/hardware/rescue-image/Makefile
index d41b79480..2254fffdf 100644
--- a/hardware/rescue-image/Makefile
+++ b/hardware/rescue-image/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=rescue-image
 PKG_VERSION:=3.6.1
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 BB_VERSION:=1.32.1
 PKG_SOURCE:=busybox-$(BB_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://www.busybox.net/downloads \
@@ -100,7 +100,7 @@ define Build/Compile
 	done
 	rm -rf "$(PKG_BUILD_DIR)/rootfs"/usr/lib/opkg
 	$(KERNEL_MAKE) $(PKG_JOBS) oldconfig $(KERNELNAME)
-	cd $(LINUX_DIR); ./usr/gen_initramfs_list.sh \
+	cd $(LINUX_DIR); ./usr/gen_initramfs.sh \
 	-o "$(PKG_BUILD_DIR)"/kernel/linux-$(LINUX_VERSION)/arch/$(LINUX_KARCH)/boot/initrd.gz -u squash -g squash \
 	"$(PKG_BUILD_DIR)/rootfs" "$(PKG_BUILD_DIR)/rootfs.extra"
 	gzip -d "$(PKG_BUILD_DIR)"/kernel/linux-$(LINUX_VERSION)/arch/$(LINUX_KARCH)/boot/initrd.gz
-- 
2.30.2

